packages/services/f-braze-adapter/src/services/BrazeConsumerRegistry.js

```diff
@@ -52,9 +51,9 @@ class BrazeConsumerRegistry {
         this.consumers = [];
 
         // subscribe to event stream for braze cards and messages
-        dispatcherEventStream.subscribe(CONTENT_CARDS_EVENT_NAME, this.applyContentCardCallbacks);
-        dispatcherEventStream.subscribe(IN_APP_MESSAGE_EVENT_NAME, this.applyInAppMessageCallbacks);
-        dispatcherEventStream.subscribe(IN_APP_MESSAGE_EVENT_CLICKS_NAME, this.applyInAppMessageClickEventsCallbacks);
+        dispatcherEventStream.subscribe(CONTENT_CARDS_EVENT_NAME, cards => this.applyContentCardCallbacks(cards));
```

Comment:
Is this change functionally equivalent to what it was before?

Reply:
Well I thought it would make no difference if it was `this.applyContentCardCallbacks` vs `cards => this.applyContentCardCallbacks(cards)`, but it was not seeing `this` within the method and throwing `consumers of undefined` error. So changing it to this makes it work, I assume some scope difference, but tbh someone else maybe able to better explain why it happens.

Reply:
Arrow functions use the value of `this` in their lexical scope which could explain it? i.e. `this` references the class instance scope rather than the callback scope?

Reply:
that makes a lot of sense actually, thanks for the explanation.